### DISCUSSION TITLE
Why are dependencies pinned?

### DISCUSSION THREAD

**Discussion by SuperchupuDev** [yao-pkg/pkg#100] [General]
It looks like a [`.yarnrc`](https://github.com/yao-pkg/pkg/blob/18ed3cfd1e79108de534d0c10e7172c9d9a89765/.yarnrc) file was added in 2018 to not use `^` in dependencies. This can lead to dependency duplication in projects. Is there any strong reason for this? It's not even consistent, some dependencies use `^` regardless

**Comment by robertsLando** [comment]
> This can lead to dependency duplication in projects. Is there any strong reason for this

I never noticed it, we can remove it as the only dep that MUST be pinned is pkg-fetch. Would you like to submit a PR as you spotted the issue? :)

**Comment by SuperchupuDev** [comment]
done in #106
